warehouse/admin/templates/admin/projects/detail.html

```diff
@@ -108,4 +108,28 @@ <h3 class="project-name text-center">{{ project.name }}</h3>
         </div>
       </div>
   </div>
+  <div class="box box-secondary">
+    <div class="box-header with-border">
+      <h3 class="box-title">Blacklist Project</h3>
+    </div>
+
+    <form method="GET" action="{{ request.route_path('admin.blacklist.add') }}">
+      <div class="box-body">
+        <div class="form-group">
```

Comment:
We no longer need this div, since this is a hidden field now it can just leave directly under the ``<form>``.